src/utils/general_utils.py

```diff
@@ -0,0 +1,30 @@
+from geopy.geocoders import Nominatim
+import time
+
+
+def geocode_address(address):
+    """This function is going to call the GPS coordinates that we specify. We can hard code it, or we can pass the
+    address from something else, like a keyboard input if that is what we want to do. Once tested I will move into
```

Comment:
Here's a good spot to utilizing a `TODO` so we can quickly find improvements to our code and easily identify them:

```suggestion
address from something else, like a keyboard input if that is what we want to do. [TODO] Once tested I will move into
```

Reply:
"NIT".